plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixConnectorTest.java

```diff
@@ -307,6 +317,70 @@ public void testMissingColumnsOnInsert()
         assertQuery("SELECT * FROM test_col_insert", "SELECT 1, 'val1', 'val2'");
     }
 
+    @Override
+    public void testTopNPushdown()
+    {
+        // Disabled since topN is not pushed down, instead partial topN is replaced with a partial Limit,
```

Comment:
nit: prefer using a SkipException. That way these actually show up as skipped tests instead of passing ones.